Add Gonzalo Peci to Team page (#1107)
sourcegraph/about: company/team/index.md

- Developer Communications
- Annapolis, Maryland, USA 🇺🇸
- [laureen@sourcegraph.com](mailto:laureen@sourcegraph.com), [LinkedIn](https://www.linkedin.com/in/laureenhudson/), [GitHub](https://github.com/LaureenH)
- Laureen is a technical and developmental editor; also a storyteller, writer, publisher, enthusiastic geek enabler, autodidact, activist, and instigator. She lives aboard a 47' catamaran with her family, and if she’s not online, it’s probably because she’s crossing an ocean.

## Christine Lovett (she/her)

- Customer Engineer
- New York, NY, USA 🇺🇸
- [christine@sourcegraph.com](mailto:christine@sourcegraph.com), [LinkedIn](https://www.linkedin.com/in/lovettchristine/), [GitHub](https://github.com/christinelovett)
- Christine is a biking, music and general fun enthusiast who likes to balance her travels exploring nature and city. Before Sourcegraph, Christine spent a few years in sales, then software development- ultimately merging her skillset in customer engingeering roles at Docker and Google. She has a strong appreciation for modern development tools and reducing friction in IT.

## Gonzalo Peci (he/him)

> - [Distribution](../../handbook/engineering/distribution/index.md) - Engineering Manager

Reviewer: I feel a little uncomfortable duplicating what teams people are on (I am fine including role here). The source of truth should be the team page and you can find out what team someone is on by doing a handbook search on their name.

Author: I agree it is not perfect, Im trying to find an easier way for people onboarding to find who does what. At the moment, I think its not easy to find:

- What teams exist? There seems to be some information in the [handbook](https://about.sourcegraph.com/handbook) but it not referenced as the source of this info or easy to get find when looking for that information. Are those all the teams?
- Who is on each team? I see now how its easy to find the team members via search but I was not obvious to me initially. I did some searches, but not all teams have a `members` section on their index so I was unable to find them, which lead me to believe it was not the "standard" way of looking for this information.

Reviewer: Engineering team documentation is here: https://about.sourcegraph.com/handbook/engineering#teams

Other functions have not yet updated the handbook to reflect their team structure. @sqs can you make this happen?

> was not obvious to me initially.

What changes could we make to to make this more obvious? This is the problem I would want to spend time solving (rather than trying to work around the problem by duplicating info on this page).

Author: > What changes could we make to to make this more obvious? This is the problem I would want to spend time solving (rather than trying to work around the problem by duplicating info on this page).

I believe having that view in the company/team/index made it easier to find the information, I can just reference this page to find works on which team doing what.

Another option would be to move https://about.sourcegraph.com/handbook index of teams to another section inside of company and reference it as the source of that information in all relevant places.

I dont think that solves the following use case:
> What team does Gonzalo work on?

As it was not clear that searching for my name was the way to find that. Maybe we can add a section on how to find that that information, or link names in the team/index.md to a search.

Author: @nicksnyder To avoid blocking this merge, I would suggest I remove the Team change for now and we can discuss that as a separate topic.

Reviewer: Yeah, let's defer trying to solve this problem. I agree it is a problem and I would like to find a way to solve it.

Author: Removed https://github.com/sourcegraph/about/commit/6d817196186a4f8b6bd5b72242df677cce5ce282.

> - Palma de Mallorca, Balearic Islands, Spain, :es:

Reviewer 2: ```suggestion
- Palma de Mallorca, Balearic Islands, Spain, 🇪🇸
```

- [gonzalo@sourcegraph.com](mailto:gonzalo@sourcegraph.com), [LinkedIn](https://linkedin.com/in/pecig), [GitHub](https://github.com/pecigonzalo), [pronounce my name 🔊](https://www.youtube.com/watch?v=4reRML9gTc4)

> - Gonzalo is originally from Argentina but has been moving around the world since 2012. He enojoys playing video games, traveling and reading. Recentrly he started playing the bass :guitar: and has a Metal/Punk garage band. He has an infrastructure background and has

Reviewer 2: ```suggestion
- Gonzalo is originally from Argentina but has been moving around the world since 2012. He enjoys playing video games, traveling and reading. Recently he started playing the bass 🎸 and has a Metal/Punk garage band. He has an infrastructure background and has
```

>   gone down the tech rabbithole so he started learning programming and other tools :nerd_face:.

Reviewer 2: ```suggestion
  gone down the tech rabbithole so he started learning programming and other tools 🤓.
```

Author: I thought those would be automatically set my the Markup interpreter, is that not the case?
